microsoft-edge/webview2/concepts/data-privacy.md

```diff
@@ -6,69 +6,117 @@ ms.author: msedgedevrel
 ms.topic: conceptual
 ms.prod: microsoft-edge
 ms.technology: webview
-ms.date: 05/16/2023
+ms.date: 06/02/2023
 ---
 # Data and privacy in WebView2
 
-The WebView2 team tracks data to keep WebView2 secure and up to date, diagnose issues, and improve WebView2. This article is for WebView2 developers.
-WebView2 developers consent to this data collection in the WebView2 Runtime Terms and Conditions License. Additionally, WebView2 follows the standards outlined in the [Microsoft Edge Privacy Whitepaper](../../privacy-whitepaper/index.md).
+The WebView2 team tracks data in order to keep WebView2 secure and up to date, diagnose issues, and improve WebView2.  WebView2 developers consent to this data collection in the WebView2 Runtime Terms and Conditions License.  To view the WebView2 Runtime Terms and Conditions License, go to [Download the WebView2 Runtime](https://developer.microsoft.com/microsoft-edge/webview2/#download-section), and then click a button, such as **Get the Link**, **Download**, or **x64**.  A dialog contains the Microsoft Software License Terms: Microsoft Edge WebView2 Runtime.

```

Comment:
@maurawinstanley - which title string to use?  Currently this passage uses two different title strings:

"the WebView2 Runtime Terms and Conditions License"
"the Microsoft Software License Terms: Microsoft Edge WebView2 Runtime"